02-microarray/differential-expression_microarray_01_2-groups.Rmd

```diff
@@ -180,10 +180,6 @@ if (!("EnhancedVolcano" %in% installed.packages())) {
   # Install this package if it isn't installed yet
   BiocManager::install("EnhancedVolcano", update = FALSE)
 }
-if (!("apeglm" %in% installed.packages())) {
```

Comment:
It runs without this now?

Reply:
Did you remove this from the Docker image and try?

Reply:
I didn't remove it from the docker image, but on my local instance I do not have it, and everything runs fine. This may be needed for the RNAseq notebook (lfcShrink uses it, I think), but not here.

Reply:
EnhancedVolcano is what complained to me. But if you are not getting that complaint, then let's remove it. It is probably RNA-seq specific then!

Reply:
Yes, `EnhancedVolcano` suggests `DESeq2`, which suggests `apeglm`, so it is probably only on the code paths related to RNAseq.